README.md

```diff
@@ -48,11 +48,11 @@ This simple application includes Ruby/Rails technology which we use at Flatstack
 
 ### Scripts
 
-* `bin/setup` - setup required gems and migrate db if needed
-* `bin/quality` - runs rubocop, brakeman, rails_best_practices and bundle-audit for the app
-* `bin/ci` - should be used in the CI or locally
-* `bin/server` - to run server locally
-* `bin/doc` - should be used to generate API documentation
+* `bin/setup` - build Docker image and prepare DB
+* `docker-compose up` - to run server locally
+* `docker-compose run app bin/rspec` - runs RSpec tests
```

Comment:
Each `run` execution will create a new container. So while development we will got multiple junky containers. It would be better to use `--rm` flag. Or use `docker-compose exec ...` (will work only if there is running container, unlike `docker-compose run ...` which works even you haven't ran `docker-compose up` before).